recipes/strainseeker/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "1.5.1" %}
+
+package:
+  name: strainseeker
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/jiangweiyao/StrainSeeker/archive/v{{ version }}.tar.gz
+  sha256: 81ace5f6d05753dcf90e84417668be3d6eb7b8d420c148f771e7e7ecc29512c1
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Is the source code available for the binaries that are shipped?

Reply:
It looks like it's on Bioconda already.
https://bioconda.github.io/recipes/genometester4/README.html

The Gitrepo is here
https://github.com/bioinfo-ut/GenomeTester4

So 
1. Make GenomeTester4 a run requirement 
2. Delete the binaries, and 
3. Build as noarch generic?

Reply:
Just found out that gmer_counter and gmer_caller (which comes in the package by requires separate make coomands) were not compiled in the Bioconda package. What is the correct way to handle this? Thank you?

Reply:
Can you add an update to the genometester4 recipe that will compile those two binaries? Yes, the others can just be removed from strainseeker and genometester4 added as a run time dependency.

Reply:
I am trying to update that package. It was on the "# OSX related errors" blacklist, and when I unblacklist it, update it, and compile it, it errors out on circleci build. I think it is OSX compiling related. What is the correct way to address this?

```
14:13:53 BIOCONDA INFO (ERR) Traceback (most recent call last):
14:13:53 BIOCONDA INFO (ERR)   File "/opt/conda/bin/conda-build", line 11, in <module>
14:13:53 BIOCONDA INFO (ERR)     sys.exit(main())
14:13:53 BIOCONDA INFO (ERR)   File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 474, in main
14:13:53 BIOCONDA INFO (ERR)     execute(sys.argv[1:])
14:13:53 BIOCONDA INFO (ERR)   File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 465, in execute
14:13:53 BIOCONDA INFO (ERR)     verify=args.verify, variants=args.variants)
14:13:53 BIOCONDA INFO (ERR)   File "/opt/conda/lib/python3.7/site-packages/conda_build/api.py", line 209, in build
14:13:53 BIOCONDA INFO (ERR)     notest=notest, need_source_download=need_source_download, variants=variants)
14:13:53 BIOCONDA INFO (ERR)   File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 2863, in build_tree
14:13:53 BIOCONDA INFO (ERR)     notest=notest,
14:13:53 BIOCONDA INFO (ERR)   File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 1994, in build
14:13:53 BIOCONDA INFO (ERR)     cwd=src_dir, stats=build_stats)
14:13:53 BIOCONDA INFO (ERR)   File "/opt/conda/lib/python3.7/site-packages/conda_build/utils.py", line 404, in check_call_env
14:13:53 BIOCONDA INFO (ERR)     return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
14:13:53 BIOCONDA INFO (ERR)   File "/opt/conda/lib/python3.7/site-packages/conda_build/utils.py", line 384, in _func_defaulting_env_to_os_environ
14:13:53 BIOCONDA INFO (ERR)     raise subprocess.CalledProcessError(proc.returncode, _args)
14:13:53 BIOCONDA INFO (ERR) subprocess.CalledProcessError: Command '['/bin/bash', '-o', 'errexit', '/opt/conda/conda-bld/genometester4_1587132765765/work/conda_build.sh']' returned non-zero exit status 2.
14:13:55 BIOCONDA ERROR COMMAND FAILED (exited with 1): /opt/conda/bin/conda build --override-channels --no-anaconda-upload -c conda-forge -c bioconda -c defaults -e /opt/conda/conda_build_config.yaml -e /opt/conda/lib/python3.7/site-packages/bioconda_utils/bioconda_utils-conda_build_config.yaml recipes/genometester4/meta.yaml

14:13:55 BIOCONDA ERROR BUILD FAILED recipes/genometester4
14:13:55 BIOCONDA INFO (COMMAND) conda build purge
14:13:55 BIOCONDA ERROR BUILD SUMMARY: of 1 recipes, 1 failed and 0 were skipped. Details of recipes and environments follow.
14:13:55 BIOCONDA ERROR BUILD SUMMARY: FAILED recipe recipes/genometester4
Error: 
Exited with code exit status 1

Step failed
Error: runner failed (exited with 101)
Task failed
Error: task failed
```

Reply:
Your options are either (1) look before the bioconda-utils traceback to debug why compilation failed or (2) add `skip:  True # [osx]` to the build section to just skip OSX.

Reply:
The GenomeTest4 binary packaged with StrainSeeker is different from the GenomeTest4 src/bin from the GenomeTest4 Version 4 release (which is the only release). The binary from the StrainSeeker repository works correctly, but the binary from GenomeTest4 can't read the database. They are also different sizes, and the one from GenomeTest4 repository is bigger and newer. I'm opening an issue with the authors, but they seem slow to respond in general. Is there any way to get it working meanwhile? Thank you.

Reply:
OK, I guess we can just ship it as is then.